WordPress/src/main/res/layout/post_prepublishing_actions_fragment.xml

```diff
@@ -8,12 +8,24 @@
         android:id="@+id/actions_recycler_view"
```

Comment:
The previous layout "fragment_prepublishing_tags" start with "fragment_" and this layout ends with "_fragment". We might want to consider sticking to one of the namings. Wdyt?

SideNote: Are `actions` and `home` interchangable? If they are, I'd consider sticking to one of them everywhere. (I'd personally prefer "home")

Reply:
Whoops. Resolved the fragment naming issue here 509857d7de0f6ae25da1a701f6e75d63b286ecfb

Did the renaming of the actions to home here 7a886d171e6548f48417b7d6bd926dba55d6f5e1